### ISSUE TITLE
Python binary in rebuild.sh

### ISSUE THREAD

**Issue by fvirdia** [fplll/g6k#25]
`rebuild.sh` seems to assume `python == python3`, which causes problems when instlaling g6k inside of Sagemath 9.0.

Personally, I believe python3 will be around as a binary for a while (probably a lot of production code is currently running with that assumption). Still, one could use something like 
```bash
pyv = $(python -c 'import sys; print(sys.version_info[:][0])') # returns 2 or 3
```
or parse the value of `python -V` (faster, but may depend on having `awk` or something like that around), to tell what `python` points to.

**Comment by malb** [comment]
How does it assume that? Does it not use whatever python it finds?

**Comment by fvirdia** [comment]
I thought g6k had dropped python2 support, my bad. Is it possible that compilation does get upset inside of `sage -sh`? Just wondering due to the reports during the tutorial at Simons.

**Comment by malb** [comment]
No idea, it is working for me here.

**Comment by lducas** [comment]
Has this maybe been solved by PR #38 ?

**Comment by malb** [comment]
I think so.
